WTWSMS/decisions/WtWSMS_religious_decisions.txt

```diff
@@ -407,57 +407,72 @@ decisions = {
 	create_priscillian_patriarchy = {
 		only_playable = yes
 		is_high_prio = yes
-	
 		potential = {
 			religion = priscillianism
 			NOT = { is_title_active = d_priscillianism }
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
 		}
-		
 		allow = {
-			is_heretic = no
 			piety = 1000
+			trait = vegetarian
+			martial = 8
+			learning = 8
```

Comment:
So? In this one, Martial is very important: the Priscillians were warlike in their resistance to the Pope, and in their attacks on Nicenes in Spain. The Priscillians also believed strongly in theology and scholarly pursuits in the name of devotion.